Formula/kertish-dfs.rb

```diff
@@ -0,0 +1,30 @@
+class KertishDfs < Formula
+  desc "Kertish FileSystem and Cluster Administration CLI"
+  homepage "https://github.com/freakmaxi/kertish-dfs"
+  url "https://github.com/freakmaxi/kertish-dfs/archive/v21.2.0066.tar.gz"
+  sha256 "1ff300a803a0d3bd068e1492319a771912cccd4ba3a72098b6d05678813af966"
+  license "GPL-3.0-only"
+  head "https://github.com/freakmaxi/kertish-dfs.git"
+
+  livecheck do
+    url :head
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "mod", "download"
+    Dir.chdir("fs-tool") { system "go", "build", "-trimpath", "-ldflags", "-X main.version=#{version}", "-o", "#{bin}/krtfs" }
+    Dir.chdir("admin-tool") { system "go", "build", "-trimpath", "-ldflags", "-X main.version=#{version}", "-o", "#{bin}/krtadm" }
+  end
+
+  def post_install
+    puts "", "", "If you need Kertish Node Executables for v#{version}, please visit https://github.com/freakmaxi/kertish-dfs/releases/tag/v21.2.0066", "", ""
+  end
```

Comment:
This is not what `post_install` methods are for. Instead, this looks like something that belongs in upstream documentation.

Reply:
In this case, I would like to ask, what is the correct way to show messages to the user? This formula is just providing CLI executables and server node files will not be there because they are not necessary to use.

Reply:
Messages for the user that aren't Homebrew-specific (i.e.. they would apply if this were packaged by some other package manager as well) should be in the software documentation (e.g. at https://github.com/freakmaxi/kertish-dfs)